res/values-ro/strings.xml

```diff
@@ -36,7 +36,7 @@
     <string name="_Score_">" Scor "</string>
     <string name="Score__">"Scor: "</string>
     <string name="Recombine_">"Recombină:  "</string>
-    <string name="WINNER_">"C ŞTIGĂTOR"</string>
+    <string name="WINNER_">"CÂŞTIGĂTOR"</string>
```

Comment:
Ş is different as Ș according to my knowledge:

Un turkic languages it's used the "Ş", while in Romanian it's used "Ș"

Reply:
Is the same Ș like into us county Caștigator(mean win)

Reply:
Winner Means Câștigător not like them make (Cstigator is wrong ) Correct is Câștigător

Reply:
I know, I am saying that it was been used "ş" instead of "ș"

Reply:
True that too

Reply:
Why i didn't get approved like others? @Jeii20
I wanna have tag translator tag! Please bro! @Jeii20

Reply:
Why @ZheiNB is approved every time and me not @Jeii20,please aploved me too, i want that Tag translator! What can i do for it Bro! @grog444